src/pages/general/login/components/loginForm/login.validation.ts

```diff
@@ -1,27 +1,18 @@
-import {BaseFormValidation} from 'lc-form-validation';
-import {requiredValidationHandler} from '../../../../../common/validations/required';
+import {
+  createFormValidation,
+  ValidationConstraints,
+  Validators,
+} from 'lc-form-validation';
 
-// TODO: Add unit tests (issues with mocking BaseFormValidation)
+const loginValidationConstraints: ValidationConstraints = {
+  fields: {
+    login: [
+      { validator: Validators.required },
+    ],
+    password: [
+      { validator: Validators.required },
+    ],
+  },
+};
 
-class LoginValidation extends BaseFormValidation {
-  public constructor() {
-    super();
-
-    this._validationEngine.initialize([
-      { formFieldName: 'login', vmFieldName: 'login' },
-      { formFieldName: 'password', vmFieldName: 'password' },
-    ]);
-
-    this._validationEngine.addFieldValidation(
-      'login',
-      requiredValidationHandler,
-    );
-
-    this._validationEngine.addFieldValidation(
-      'password',
-      requiredValidationHandler,
-    );
-  }
-}
-
-export const loginValidation = new LoginValidation();
+export const loginFormValidation = createFormValidation(loginValidationConstraints);
```

Comment:
After discussion, you are right these are not unit tests, are more  black box tests / assembly, but I find them quite useful, that was something that I missed on the Angular 1 validation, I just want to check that this validation group behaves well feeding cases, not worrying about wiring up components. Only thing to consider, when we have async validations we should mock them, that's not the case.